ciao-deploy/deploy/utils.go

```diff
@@ -47,6 +47,32 @@ func DefaultImageCacheDir() string {
 	return path.Join(u.HomeDir, ".cache", "ciao", "images")
 }
 
+// CleanupImages will remove all images from the cache directory that
+// match a specific pattern. Images that match this pattern but should
+// be preserved can be specified in the "keep" slice.
+func CleanupImages(pattern string, keep []string, imageCacheDir string) error {
+	imagePath := fmt.Sprintf("%s/%s", imageCacheDir, pattern)
+
+	matches, err := filepath.Glob(imagePath)
+	if err != nil {
+		return errors.Wrap(err, "Unable to clean old images")
+	}
+
+	keepMap := make(map[string]bool)
+	for _, k := range keep {
+		keepMap[k] = true
+	}
+
+	for _, m := range matches {
+		if keepMap[m] {
+			continue
+		}
+
+		os.Remove(m)
```

Comment:
We should discard the error here.  Right now ciao-deploy is errcheck clean but this will introduce a warning from this tool.

Reply:
thanks - fixed.